.evergreen/config.yml

```diff
@@ -349,6 +459,57 @@ tasks:
     - func: "test python"
       vars: { variant_name: windows-test }
 
+- name: "release-python-macos-1014"
+  tags: ["release_python_tag"]
+  run_on: macos-1014
+  commands:
+    - func: "fetch source"
+    - func: "build python release macos-1014"
+    - func: "test python release"
+      vars: { PYTHON: /Library/Frameworks/Python.framework/Versions/3.7/bin/python3 }
+    - func: "upload python release"
+
+- name: "release-python-macos-1100"
+  tags: ["release_python_tag"]
+  run_on: macos-1100
+  commands:
+    - func: "fetch source"
+    - func: "build python release macos-1100"
+    - func: "test python release"
+      vars: { PYTHON: /Library/Frameworks/Python.framework/Versions/3.10/bin/python3 }
+    - func: "upload python release"
+
+- name: "release-python-linux"
+  tags: ["release_python_tag"]
+  run_on: ubuntu2004-large
+  exec_timeout_secs: 216000  # 60 minutes (manylinux task is slow).
+  commands:
+    - func: "fetch source"
+    - func: "build python release"
+    - func: "test python release"
+      vars: { PYTHON: python }
+    - func: "upload python release"
+
+- name: "release-python-windows"
+  tags: ["release_python_tag"]
+  run_on: windows-test
```

Comment:
This should be windows-64-vsMulti-small.

Reply:
Done